package.json

```diff
@@ -7,11 +7,15 @@
     "react-dom": "^16.9.0",
     "react-scripts": "0.9.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "big-js": "^3.1.3",
+    "node-sass": "^4.12.0"
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "sass": "sass --watch src/styles/scss:src/styles/css"
```

Comment:
Why did we need to include this script? Shouldn't `sass` automatically get updated by default whenever there's a change in any styling?